crates/telio-sockets/src/protector/apple.rs

```diff
@@ -334,17 +334,27 @@ fn get_primary_interface_names() -> Vec<String> {
             let primary_service_dictionary = store
                 .get(format!("State:/Network/Service/{}/IPv4", service).as_str())
                 .and_then(CFPropertyList::downcast_into::<CFDictionary>);
+
             if let Some(primary_service_dictionary) = primary_service_dictionary {
-                let interface_name = primary_service_dictionary
-                    .find(unsafe { schema_definitions::kSCPropInterfaceName }.to_void())
-                    .map(|ptr| unsafe { CFString::wrap_under_get_rule(*ptr as CFStringRef) });
-                if let Some(interface_name) = interface_name {
-                    primary_interface_names.push(interface_name.to_string());
+                if primary_service_dictionary
+                    .find(unsafe { schema_definitions::kSCPropNetIPv4Router }.to_void())
+                    .is_some()
+                {
+                    let interface_name = primary_service_dictionary
+                        .find(unsafe { schema_definitions::kSCPropInterfaceName }.to_void())
+                        .map(|ptr| unsafe { CFString::wrap_under_get_rule(*ptr as CFStringRef) });
+                    if let Some(interface_name) = interface_name {
+                        primary_interface_names.push(interface_name.to_string());
+                    }
                 }
             }
         }
     }
 
+    telio_log_info!(
```

Comment:
optional: make debug log

Reply:
:thinking: Hmm, I suspect this should not be super spammy, and `debug` will not appear in real usage. Only in debug builds, which may cause finding issues with interface selection harder...

Reply:
I have chatted with Marcin, and they too are kinda in favor of `Info`, so I will leave it as `Info`.